hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/snapshot/PrintCompactionLogDagHandler.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.shell.snapshot;
+
+import org.apache.hadoop.ozone.client.OzoneClient;
+import org.apache.hadoop.ozone.shell.Handler;
+import org.apache.hadoop.ozone.shell.OzoneAddress;
+import picocli.CommandLine;
+
+import java.io.IOException;
+
+/**
+ * Handler to generate image for current compaction DAG in the OM.
+ * ozone sh snapshot print-log-dag.
+ */
+@CommandLine.Command(name = "print-log-dag",
+    aliases = "pld",
+    description = "List snapshots for the buckets.")
+public class PrintCompactionLogDagHandler extends Handler {
+
+  @CommandLine.Option(names = {"-f", "--fileName"},
+      description = "Name of the image file. (optional)")
+  private String fileName;
+
+  @CommandLine.Option(names = {"-gt", "--graphType"},
```

Comment:
`-t`, or `--gt`

```suggestion
  @CommandLine.Option(names = {"-t", "--graphType"},
```

Reply:
In addition, please avoid camelCase, use `--graph-type`, etc.

Reply:
Changed as suggested.